app/models/notification/right_side_popup.rb

```diff
@@ -0,0 +1,9 @@
+class Notification::RightSidePopup < Notification
+  def self.notify(user, course, options = {})
+    notification = RightSidePopup.new
+    notification.user_id = user.id
+    notification.course_id = course.id
```

Comment:
These 3 lines can be combined: <code> RightSidePopup.new(user: user ,course: course) </code>